presto-docs/src/main/sphinx/release/release-0.183.rst

```diff
@@ -0,0 +1,60 @@
+=============
+Release 0.183
+=============
+
+General Changes
+---------------
+
+* Fix planning failure for queries that use ``GROUPING`` and contain aggregation expressions
+  that require implicit coercions.
+* Fix planning failure for ``LEFT JOIN`` converted to ``INNER JOIN`` without equi-conditions.
+* Fix issue where a query may have a reported memory that is higher than actual usage when
+  an aggregation is followed by other non-trivial work in the same stage. This can lead to failures
+  due to query memory limit, or lower cluster throughput due to perceived insufficient memory.
+* Fix formatting in ``EXPLAIN ANALYZE`` output.
+* Improve error message when query contains an unsupported correlated subquery.
+* Improve performance of ``CAST(json_parse(...) AS ...)``.
+* Add :func:`map_from_entries` and :func:`map_entries` functions.
+* Make it possible for `CHAR` functions (e.g. :func:`substr`) to return a value that has
```

Comment:
What does "make it possible" mean? Were the functions previously returning incorrect results? Would they throw an error? i.e., should this say "Fix a correctness issue for ..."